scripts/ci-entrypoint.sh

```diff
@@ -132,14 +132,9 @@ create_cluster() {
 }
 
 install_addons() {
-    # Get CCM cluster CIDRs from Cluster object if not set
-    if [[ -z "${CCM_CLUSTER_CIDR:-}" ]]; then
-        CCM_CLUSTER_CIDR=$(${KUBECTL} get cluster "${CLUSTER_NAME}" -o=jsonpath='{.spec.clusterNetwork.pods.cidrBlocks[0]}')
-        if CIDR1=$(${KUBECTL} get cluster "${CLUSTER_NAME}" -o=jsonpath='{.spec.clusterNetwork.pods.cidrBlocks[1]}' 2> /dev/null); then
-            CCM_CLUSTER_CIDR="${CCM_CLUSTER_CIDR:-}\,${CIDR1}"
-        fi
-    fi
-    echo "CCM cluster CIDR: ${CCM_CLUSTER_CIDR:-}"
+    # Get cluster CIDRs from Cluster object
+    CIDR0=$(${KUBECTL} get cluster "${CLUSTER_NAME}" -o=jsonpath='{.spec.clusterNetwork.pods.cidrBlocks[0]}')
+    CIDR1=$(${KUBECTL} get cluster "${CLUSTER_NAME}" -o=jsonpath='{.spec.clusterNetwork.pods.cidrBlocks[1]}' 2> /dev/null)
```

Comment:
This is a graceful command if there is only one CIDR in `.spec.clusterNetwork.pods.cidrBlocks`?

Reply:
correct

Reply:
actually let me double check that

/hold

Reply:
I got it wrong before, needed the `|| true` as well otherwise it fails the script (even if stderr is suppressed)

Reply:
/hold cancel